### ISSUE TITLE
Questions related to rendering and sensor observations.

### ISSUE THREAD

**Issue by hercky** [kngwyu/mujoco-maze#7]
Hello! 

I have two questions:

1. How is it possible to get the top view of the maze as shown in the README. For instance, when I render the environment I get something like this instead:
![maze_view](https://user-images.githubusercontent.com/1503597/119057904-54e07e00-b99b-11eb-8585-155cc6c597d6.png)
I was wondering if you can share the snippet for how the images in readme were rendered.

2. Why aren't the range sensor used for the observation space as done in the reference [tensorflow/models](https://github.com/tensorflow/models) repo over [here](https://github.com/tensorflow/models/blob/a570167f14d36fd4e1c740d68c3e6dd92ed72436/research/efficient-hrl/environments/maze_env.py#L323)? Shouldn't that affect the learning too? 

PS: Thanks for releasing this package. This is pretty cool!

**Comment by kngwyu** [comment]
> How is it possible to get the top view of the maze as shown in the README

I manually moved the camera using mujoco-py's GUI. I am also curious if we can move the camera by some snippets, and I'll let you know when I succeed.

> Why aren't the range sensor used for the observation space as done in the reference tensorflow/models repo over here? Shouldn't that affect the learning too?

I removed it since it was not used in experiments.
First, let's confirm that `range_sensor_obs` is an empty numpy array unless `n_bins > 0` here: https://github.com/tensorflow/models/blob/238922e98dd0e8254b5c0921b241a1f5a151782f/research/efficient-hrl/environments/maze_env.py#L377.
Then let's see https://github.com/tensorflow/models/blob/238922e98dd0e8254b5c0921b241a1f5a151782f/research/efficient-hrl/environments/create_maze_env.py, and take care of `n_bins`. We can see that `n_bins` is `8` if the envname starts with `Ego` and otherwise `0`. However, then we can see an if block like
```python
if envname == "Point":
elif ...
```
, and here we cannot see `Ego`.
So, I judged the egocentric view is a duplicated feature.

But, I think the egocentric vs topdown view is an interesting question in RL, and retrieving the feature in some way would be worth considering.


Thank you for your questions, and let me know if you have any other questions.
